### PR TITLE
App extension improvements

### PR DESCRIPTION
**Pull request by aramsargsyan**

### CODE REVIEW

File: MatrixKit/Views/RoomInputToolbar/MXKRoomInputToolbarView.m [hunk 1]
```diff
@@ -357,93 +353,6 @@ - (void)pasteText:(NSString *)text
 
 #pragma mark - MXKImageCompressionSize
```
Comment (giomfo) [inline MatrixKit/Views/RoomInputToolbar/MXKRoomInputToolbarView.m]: Remove this #pragma mark, or maybe move it to MXKTools

File: MatrixKit/Views/MXKPieChartHUD.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
+//
```
Comment (giomfo) [inline MatrixKit/Views/MXKPieChartHUD.h]: Set the correct file header: 
"/*
 Copyright 2017 Aram Sargsyan
[...]
*/"

File: MatrixKit/Views/MXKPieChartHUD.h [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+//  MXKPieChartHUD.h
+//  Pods
+//
+//  Created by Aram Sargsyan on 8/15/17.
+//
+//
+
+#import <UIKit/UIKit.h>
+
+@interface MXKPieChartHUD : UIView
```
Comment (giomfo) [inline MatrixKit/Views/MXKPieChartHUD.h]: Please inherit the MXKView class (required to handle UI theme change)

File: MatrixKit/Views/MXKPieChartHUD.m [hunk 4]
```diff
@@ -0,0 +1,93 @@
+//
+//  MXKPieChartHUD.m
```
Comment (giomfo) [inline MatrixKit/Views/MXKPieChartHUD.m]: Set the correct file header

File: MatrixKit/Views/MXKPieChartHUD.m [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- (void)configureFromNib
+{
+    NSBundle *bundle = [NSBundle mxk_bundleForClass:self.class];
+    [bundle loadNibNamed:NSStringFromClass(self.class) owner:self options:nil];
+    self.hudView.frame = self.bounds;
+    
+    self.clipsToBounds = YES;
+    self.layer.cornerRadius = 10.0;
+    
+    self.pieChartView.backgroundColor = [UIColor clearColor];
```
Comment (giomfo) [inline MatrixKit/Views/MXKPieChartHUD.m]: You should override [MXKView customizeViewRendering] method in order to move 
the color settings in it.

**Comment by aramsargsyan** [comment]
Signed-off-by: Aram Sargsyan aram.sargsyan.1997@gmail.com

**Pull request merged**